cli/src/services/crawl.service.spec.ts

```diff
@@ -0,0 +1,235 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
We should get rid of these I think.

Reply:
You should not be using `require()` to import a module.. Can you not get it to work with `import * as matchers from 'jest-extend'`?

Reply:
jest-extend seems to be very special. Even with that import line I get 

    Cannot find module 'jest-extend' or its corresponding type declarations.ts(2307)